demo/index.html

```diff
@@ -40,6 +48,22 @@
 
     });
 
+    Polymer({
+      is: 'type-one',
+
+      ready: function() {
+        this.textContent = new Polymer.IronMetaQuery({type: "type1"}).list;
```

Comment:
Consider calling `JSON.stringify` on the list to make it clearer to the reader that it's a list.